### PR TITLE
feature(elm): Display timer

### LINKED ISSUE #109: feat(elm): Add timer to questions
**Type**

Feature

blocked_by: #148 

**Description**
After introducing #108 we will need to tell users how much time they have until the question changes. 

This issue's goal is only to visualize sort of a clock that would let players know how much time they have left. 

The issue taking care of ws synchronization: #148

### PR DESCRIPTION
**Pull request by mrapacz**
**Summary**
This PR closes #109 

**Related issues**
closes: #109  

**Test plan**
manual

recording

### CODE REVIEW

File: aion/web/elm/src/Update.elm [hunk 1]
```diff
@@ -468,10 +471,14 @@ update msg model =
                 )
 
         ReceiveDisplayQuestion _ ->
-            { model | roomState = QuestionDisplayed } ! []
+            { model
+                | roomState = QuestionDisplayed
+                , progressBar = model.progressBar |> withProgress 0 |> withRunning Uninitialized |> withStart 0
```
Comment (pmrukot) [inline aion/web/elm/src/Update.elm:476]: Is there a way to format pipes in new line each?

File: aion/web/elm/src/Update.elm [hunk 2]
```diff
@@ -468,10 +471,14 @@ update msg model =
...
+            }
+                ! [ Delay.after progressBarDelay millisecond Tick ]
 
         ReceiveQuestionBreak raw ->
-            { model | roomState = QuestionBreak } ! []
+            { model | roomState = QuestionBreak, progressBar = model.progressBar |> withRunning Stopped |> withProgress 0 } ! []
```
Comment (pmrukot) [inline aion/web/elm/src/Update.elm]: Kind of inconsistent formatting, above you have `| roomState = ...` and `progressBar = ...` in separate lines.

**Review by pmrukot — APPROVED** [review]
Looks good, code and the bar itself.

**Comment by pmrukot** [comment]
Also tested it locally. Im wondering if during the break we should display empty bar or full - I would go for full. This is relatively minor thing but bar going instantly empty after filling up makes it look slightly weird/glitchy.

**Comment by mrapacz** [comment]
@pmrukot I asked myself the same question. Then again after the question break it looked kinda glitchy going instantly from 100% to 0 :(

I think we could also display a label on the progress bar during the question break (or I could make it striped to symbolise that it's locked in a way). What do you think is the best solution?

File: aion/web/elm/src/Update.elm (hunk 1 above)
Comment (mrapacz) [reply aion/web/elm/src/Update.elm:476]: Absolutely, I could do it, I wanted to refrain from building a visual tree and update each part of the model in one line. I can make it multiline though

File: aion/web/elm/src/Update.elm (hunk 2 above)
Comment (mrapacz) [reply aion/web/elm/src/Update.elm]: True, I would switch to one model-entry per line, like above. Does that sound ok?

**Comment by mrapacz** [comment]
@pmrukot the bar stops and becomes striped and animated on questionBreak.

**Pull request merged**